share/spack/gitlab/cloud_pipelines/stacks/aws-pcluster-neoverse_v1/setup-pcluster.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env bash
```

Comment:
Could we move this to `share/spack/gitlab/cloud_pipelines/scripts/pcluster/setup-pcluster.sh`?